frontend/src/metabase/visualizations/components/settings/ChartNestedSettingSeriesSingle.jsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+
+import { getAccentColors } from "metabase/lib/colors/groups";
+import ColorSelector from "metabase/core/components/ColorSelector";
+import InputBlurChange from "metabase/components/InputBlurChange";
+
+// various props injected by chartSettingNestedSettings HOC
+export default class ChartNestedSettingSeries extends React.Component {
```

Comment:
Since this file is new should we create the component using a function instead of a class component?

Reply:
You're right, I got lazy haha. I'll update this :)